### ISSUE TITLE
bad credential with Authorization

### ISSUE THREAD

**Issue by ureyni** [lexik/LexikJWTAuthenticationBundle#586]
Hi, 
i working on symfony 4.1 with Lexik package.
i get token , no problem this progress. but when i send token with Authorization header tag's value 
it give error  {"code":401,"message":"Bad credentials" }". 
i try firefox RestClient and Jquery "beforeSend: function(xhr, settings) { xhr.setRequestHeader('Authorization','Bearer ' + localStorage.getItem("authorization_token") ); },"
both not successful. 
when i comprehensive debug , i saw , i cant send "Authorization" header tag and value.
there is not "Authorization" header tag/value in $request->header array.

i don't understand this .

when i change configuration in lexik_jwt_authentication.yaml
        authorization_header:      # look for a token as Authorization Header
            enabled: true
            prefix:  Bearer
            name:    AuthorizationJzm

and send data with  "AuthorizationJzm" header tag and value 
not problem.
what am i missing ?

**Comment by chalasr** [comment]
Hey, I guess you're using apache which suffers from [this issue](https://github.com/lexik/LexikJWTAuthenticationBundle/blob/master/Resources/doc/index.md#important-note-for-apache-users), isn't it? Does the suggested fix help?

**Comment by ureyni** [comment]
Yes I using Apache and this configuration be solve of my problem.
thank you very much.
what do you think about hazelcast or redis can be store for token.
